game/lua/util/dump-rules.lua

```diff
@@ -46,22 +46,34 @@ local typeAreas =
 
 local typeAreaOrder =
 {
-  "Weapons",
-  "Bullets",
-  "Warheads",
   "Infantry",
   "Units",
   "Aircraft",
-  "Buildings"
+  "Buildings",
+  "Weapons",
+  "Bullets",
+  "Warheads"
 }
 
 local outputFileName = "RULES-DEFAULT.INI"
 local rulesFile = nil
 
 local function dumpRulesForTypeArea(rulesFile, typeAreaName, typeArea)
   rulesFile:write(string.format("## %s\n", typeAreaName))
+  rulesFile:write("\n")
+
+  rulesFile:write(string.format("[%s]\n", typeAreaName))
+  idx=1
+  log("Creating list %s", typeAreaName)
+  for _, areaType in ipairs(typeArea.getTypes()) do
+    rulesFile:write(string.format("%s=%s\n",tostring(idx),areaType))
```

Comment:
Need to check here if the `areaType` is a mod type - if so we don't want to dump it (otherwise we leak test mods into the default rules)

Reply:
Already fixed in my local branch.